06_gpu_and_ml/comfyui/comfyapp.py

```diff
@@ -72,126 +48,105 @@
     .run_commands(  # use comfy-cli to install the ComfyUI repo and its dependencies
         "comfy --skip-prompt install --nvidia"
     )
-    .add_local_file(
+    .add_local_file(  # copy ComfyUI workflow JSON to the container
         Path(__file__).parent / "workflow_api.json",
         "/root/workflow_api.json",
         copy=True,
```

Comment:
We don't need `copy=True` here if we move this step to the end of image build since we don't use it in the subsequent steps. If we remove `copy=True` and set it as the last step of the image build we will avoid rebuilding subsequent image layers when we modify the workfow file